### PR TITLE
CHK-327: Add setManualPrice mutation

### PR DESCRIPTION
**Pull request by jeffersontuc**
#### What problem is this solving?

This PR uses and exposes the `SetManualPrice` mutation. 

#### How to test it?

Follow the test plan described on [product-list#79](https://github.com/vtex-apps/product-list/pull/79)

#### Screenshots or example usage:

image

image

#### Related to / Depends on

[checkout-graphql#89](https://github.com/vtex/checkout-graphql/pull/89)
[checkout-resources#55](https://github.com/vtex-apps/checkout-resources/pull/55)
[product-list#79](https://github.com/vtex-apps/product-list/pull/79)

### CODE REVIEW

File: react/OrderItems.tsx [hunk 1]
```diff
@@ -271,6 +274,24 @@ const useAddItemsTask = (
   return addItemTask
 }
 
+const useSetManualPrice = () => {
+  const [mutateSetManualPrice] = useMutation<SetManualPrice>(SetManualPrice)
+
+  const setManualPriceTask = (price: number, itemIndex: number) => {
```
Comment (lucasecdb) [inline react/OrderItems.tsx:280]: can you wrap this function inside a `useCallback`?

File: react/OrderItems.tsx [hunk 2]
```diff
@@ -570,15 +596,20 @@ export const OrderItemsProvider: FC = ({ children }) => {
     [enqueueTask, setOrderForm, updateItemsTask]
   )
 
+  const setManualPrice = (price: number, itemIndex: number) => {
```
Comment (lucasecdb) [inline react/OrderItems.tsx:599]: same as above

**Review by lucasecdb — COMMENTED** [review]
LGTM! can you fix the lint issues? I think if you run `yarn lint --fix` in the root folder should be good

**Review by lucasecdb — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+
```
Comment (wdsrocha) [inline CHANGELOG.md:10]: nit to keep consistency
```suggestion
```

**Review by wdsrocha — APPROVED** [review]

**Review by lucasecdb — CHANGES_REQUESTED** [review]
Tests are failing, can you take a look?

**Comment by jeffersontuc** [comment]
> Tests are failing, can you take a look?

Yeah, I'm looking at it! Thanks!

**Comment by jeffersontuc** [comment]
> Tests are failing, can you take a look?

Apparently, it can't find the `SetManualPrice` mutation. I'll wait some minutes after trying to re-run again, since I just  deployed the version of the `checkout-resources` which adds this mutation.

**Comment by lucasecdb** [comment]
> Apparently, it can't find the `SetManualPrice` mutation. I'll wait some minutes after trying to re-run again, since I just deployed the version of the `checkout-resources` which adds this mutation.

I think you need to mock it, no? since the package toolbelt injects here is only for the typings

**Comment by jeffersontuc** [comment]
> > Apparently, it can't find the `SetManualPrice` mutation. I'll wait some minutes after trying to re-run again, since I just deployed the version of the `checkout-resources` which adds this mutation.
> 
> I think you need to mock it, no? since the package toolbelt injects here is only for the typings

I do, indeed! Thanks

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**